tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -25,7 +25,7 @@ raw_amp = <span>foo & bar</span>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 many_nbsp = <div>&nbsp; &nbsp; </div>;
 single_nbsp = <div>&nbsp;</div>;
-many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
+many_raw_nbsp = <div> </div>;
```

Comment:
&nbsp; are non breaking spaces. They are special spaces that are not collapsable. So they should remain. Something is either wrong with the parser or this implementation.